examples/axios.test.js

```diff
@@ -883,4 +883,19 @@ describe('axios', function() {
       console.log(res3);
     }));
   });
+  it('axios-dataA', async function() {
+    await axios.get('https://httpbin.org/get').then((res) => {
+      res.data; // Returns the HTTP response body at the server we requested.
+      typeof res.data; // object
+    });
+  });
+  it('axios-dataB', async function() {
+    await axios.get('https://httpbin.org/get', {
+      responseType: 'text',
+      transformResponse: [data => data] // the name of the variable does not matter
```

Comment:
I'd double check one more time that `transformResponse` is necessary, it doesn't seem like it should be but I could be wrong